doc/tutorials/enable_ivshmem.rst

```diff
@@ -0,0 +1,170 @@
+.. _enable_ivshmem:
+
+Enable Inter-VM Communication Based on ``ivshmem``
+##################################################
+
+You can use inter-vm communication based on the ``ivshmem`` dm-land
+solution or hv-land solution, according to the usage scenario needs.
+(See :ref:`ivshmem-hld` for a high-level description of these solutions.)
+While both solutions can be used at the same time, VMs using different
+solutions can not communicate with each other.
+
+ivshmem dm-land usage
+*********************
+
+Add this line as an ``acrn-dm`` boot parameter::
+
+     -s slot,ivshmem,shm_name,shm_size
+
+where
+
+-  ``-s slot``  - Specify the virtual PCI slot number
+
+-  ``ivshmem``  - Virtual PCI device name
+
+-  ``shm_name`` - Specify a shared memory name. Post-launched VMs with the same
+   ``shm_name`` share a shared memory region. The ``shm_name`` needs to start
+   with ``dm:/`` prefix. For example, ``dm:/test``
+
+-  ``shm_size`` - Specify a shared memory size. The unit is byte. The size
```

Comment:
Not related to our documentation, but it would be more user-friendly to use MB as the default unit. It would make for smaller and more readable numbers and we really don't need the level of granularity offered by using bytes as the unit.

Actually, a question just popped in my mind: what happens of the user messes up the number and does not use an exact multiple of 2MB in the command-line? Have we tested that?

Reply:
Yes, I agree with you，actually the MB as the default unit is our current design, I will refine this from doc, configuration tool and acrn-dm code.

For hv-land, ACRN offline tool will check the user's input, and if the input does not meet the requirements, it will indicate to user.
For dm-land, acrn-dm will do the check.